libvast/include/vast/format/zeek.hpp

```diff
@@ -251,6 +251,7 @@ class reader final : public single_schema_reader {
   std::string unset_field_;
   vast::module module_;
   type schema_;
+  type output_schema_;
```

Comment:
Maybe add a comment that this is only here temporarily until the `cast` operator is available?

Reply:
The readers need to change so much for the from/read refactoring that I think we are likely to clone them anyways. I don't think we can plug the current reader API into the upcoming framing, so we'll pick out what we need then.